integration-cli/docker_cli_inspect_test.go

```diff
@@ -92,14 +83,9 @@ func (s *DockerSuite) TestInspectTypeFlagContainer(c *check.C) {
 	dockerCmd(c, "run", "--name=busybox", "-d", "busybox", "top")
 
 	formatStr := fmt.Sprintf("--format='{{.State.Running}}'")
-	out, exitCode, err := dockerCmdWithError("inspect", "--type=container", formatStr, "busybox")
-	if exitCode != 0 || err != nil {
-		c.Fatalf("failed to inspect container: %s, %v", out, err)
-	}
-
-	if out != "true\n" {
-		c.Fatal("not a container JSON")
-	}
+	out, exitCode := dockerCmd(c, "inspect", "--type=container", formatStr, "busybox")
+	c.Assert(exitCode, checker.Equals, 0)
```

Comment:
I think this check isn't needed since if an error happens then exitCode is always not zero and dockerCmd would have failed first